### PR TITLE
changed jwtBuilder attribute elapsedNbf to nbfOffset

### PR DESCRIPTION
**Pull request by mmerali96**
#11868

### CODE REVIEW

**Comment by skasund** [comment]
L2 message review not required

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -101,9 +101,9 @@ jwkEndpointUrl.desc=Specifies a JWK end point URL.
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* and javax* system properties.
 
-elapsedNBF=Elapsed not before claim
-elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted for processing. The value must be a NumericDate. 
+nbfOffset=Not before claim offset
+nbfOffset.desc=Identifies the time when JWT will start to be accepted for processing. The value must be a NumericDate.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties:105]: Suggested update

>Identifies the time when JWT is accepted for processing. The value must be a NumericDate object.

File: dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -101,9 +101,9 @@ jwkEndpointUrl.desc=Specifies a JWK end point URL.
...
-elapsedNBF=Elapsed not before claim
-elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted for processing. The value must be a NumericDate. 
+nbfOffset=Not before claim offset
+nbfOffset.desc=Identifies the time when JWT will start to be accepted for processing. The value must be a NumericDate. 
 
 #Do not translate "amr"
 amrValues=Authentication methods references in string array
-amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
\ No newline at end of file
+amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties:109]: Update

>Specify a comma separated list of valid amr claim values to be used.

To

>Specify a comma-separated list of valid amr claim values to be used.

**Review by Charlotte-Holt — COMMENTED** [review]
@mmerali96 I added a couple of ID review comments. Let me know if you have questions.

**Comment by mmerali96** [comment]
@Charlotte-Holt I made those changes. Realized I created this PR in the wrong fork so i remade it under my fork. You can find the new PR here:  #14112

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#run-libby-bot

**Comment by mmerali96** [comment]
#build after merge conflict

**Comment by mmerali96** [comment]
#build one last build

**Comment by mmerali96** [comment]
#run-libby-bot

**Comment by mmerali96** [comment]
#run-libby-bot

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#run-libby-bot

**Comment by mmerali96** [comment]
#run-libby-bot

**Pull request merged**